packages/console-types/src/types/endpoints.ts

```diff
@@ -347,8 +347,9 @@ export const endpoints = {
 
 // This type is required since Endpoints cannot parse if/then/else since it is too deep
 export type Endpoints = Record<string,
-FastDataProjectionEndpoint |
-CrudEndpoint |
-SingleViewEndpoint |
-GenericEndpoint
+  FastDataProjectionEndpoint |
+  CrudEndpoint |
+  SingleViewEndpoint |
+  CustomEndpoint |
```

Comment:
isn't the CustomEndpoint already included in the generic one?

Reply:
It doesn't, since the CustomEndpoint (created with #336 ) includes the fields `port` and `useDownstreamProtocol`.